src/StmtToHTML.cpp

```diff
@@ -2528,11 +2534,20 @@ class PipelineHTMLInspector {
         std::ifstream assembly;
         assembly.open(asm_file.c_str());
 
-        // Slurp the code into asm_stream
+        // Slurp the code into asm_stream...
+        // Very inefficiently...
+        std::ostringstream stream;
         std::string line;
         while (getline(assembly, line)) {
-            asm_stream << line << "\n";
+            if (line.length() > 500) {
+                // Very long lines in the assembly are typically the _gpu_kernel_sources
+                // or other buffers (such as model weights) as a raw ASCII block in the
```

Comment:
I would say something more realistic like "static LUTs" here... model weights are so large, they should be `ImageParams`.